Sources/Commands/Workspace.swift

```diff
@@ -83,20 +87,27 @@ public class Workspace {
         /// The current version of the dependency, if known.
         public let currentVersion: Version?
 
+        /// The dependency is being edited.
+        let isEdited: Bool
```

Comment:
I think `isInEditableState` would be more clear. "edited" within the codebase seems a little unambiguous. If you think that is too verbose, I'd use `isEditable` at least.

Also, I think this doc comment should explain a tiny bit about what that actually means.